test/testcafe/spec/TerminalView_spec.js

```diff
@@ -140,8 +141,11 @@ async function setup(t) {
   test
     .requestHooks(mock)(testTitle, async t => {
       const terminalViewPage = await setup(t);
-      await t.expect(await terminalViewPage.goBackLinkExists()).notOk();
```

Comment:
I did have to make a small change here

Reply:
for some reason i am failing parity-v3 test on bacon, but I am passing locally

Reply:
In v2, are these links the same (as far as the text they display)? is the only difference between the two the classes attached to the links?

Reply:
Yes the text they display are the same just the classes are different

![Screen Shot 2022-10-05 at 11 15 32 AM](https://user-images.githubusercontent.com/107433508/194136898-87d9de
![Screen Shot 2022-10-05 at 11 34 41 AM](https://user-images.githubusercontent.com/107433508/194136908-052dce34-34f7-4cba-b9f9-2b80d23791b7.png)
e5-1907-416f-ae43-c5f6200038cc.png)

Reply:
yeah turns out the text for those links are the same.  But the data-se's are different.
![Screen Shot 2022-10-05 at 12 20 05 PM](https://user-images.githubusercontent.com/107433508/194145277-518815f7-765d-4a9a-8aa5-666617a92575.png)
![Screen Shot 2022-10-05 at 11 34 41 AM](https://user-images.githubusercontent.com/107433508/194145281-8dee8b31-a92b-4168-a587-a27e5340530f.png)